packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1050,42 +1046,42 @@ curl -v -X PUT \
 "https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/KVJUKUS7IFCE2SKO/targets/catalog/apps/amazon_aws/0oasrudLtMlzAsTxk0g3"
 ```
 
-###### Response Example
+###### Response example
 
 
 ``` http
 HTTP/1.1 204 No Content
 ```
 
-##### Add App Instance Target to App Administrator Role given to a Group
+##### Add an App Instance target to an App administrator role given to a Group
 
 
 <ApiOperation method="put" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/catalog/apps/${appName}/${appInstanceId}" />
 
-Adds an app instance target for an `APP_ADMIN` role assigned to a group
+Adds an App Instance target for an `APP_ADMIN` Role assigned to a Group
 
-Adding the first app or (app instance) target changes the scope of the role assignment from applying to all app targets to applying to the specified target.
+When you add the first App or App Instance target, you reduce the scope of the role assignment. The Role no longer applies to all App targets, but applies only to the specified target.
 
-App Targets and App Instance Targets cannot be mixed for the same app name. For example, you cannot specify that an administrator has access to manage Salesforce (the entire app type) and specific instances of the Salesforce app; it must be one or the other.
+> **Note:** You can target a mixture of both App and App Instance targets, but can't assign permissions to manage all instances of an App and then a subset of that same App. For example, you can't specify that an admin has access to manage all instances of a Salesforce app and then also specific configurations of the Salesforce app.
```

Comment:
FYI, we have this gap as well: https://oktainc.atlassian.net/browse/OKTA-269111 not sure if you want to address it with this same change

Reply:
That applies to the Product Help site, which is a totally different repo, doc system, etc.